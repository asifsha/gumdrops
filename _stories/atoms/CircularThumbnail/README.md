The `<CircularThumbnail>` component is used to display images within a circular frame. Recommended props to pass in are src="https://domain.com/yourimage.png" and alt="name". Any other props that are passed in are also accepted.
