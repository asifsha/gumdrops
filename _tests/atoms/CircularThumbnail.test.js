/* globals mount */
import React from 'react';
import CircularThumbnail from '../../components/atoms/CircularThumbnail';

const defaultProps = {
    size: 'sm',
    context: 'danger',
    className: 'foo-class'
};

describe('Expect <CircularThumbnail>', () => {
    it('to render', () => {
        const wrapper = mount(<CircularThumbnail {...defaultProps} />);
        expect(wrapper).toMatchSnapshot();
    });
});
