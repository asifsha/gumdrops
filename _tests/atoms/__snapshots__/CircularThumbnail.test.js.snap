// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect <CircularThumbnail> to render 1`] = `
<CircularThumbnail
  className="foo-class"
  context="danger"
  size="sm"
>
  <img
    className="gds-circular-thumbnail foo-class gds-circular-thumbnail--danger gds-circular-thumbnail--sm"
  />
</CircularThumbnail>
`;
